server/routes/nav.ts

```diff
@@ -2,7 +2,7 @@ import { Request } from 'express';
 
 export const home = { title: 'Home', href: '/' };
 export const tutorSessions = { title: 'Tutor Sessions', href: '/tutor-sessions' };
-export const profile = { title: 'Profile', href: '/profile' }
+export const profile = { title: 'Profile', href: '/profile/:userId' }
```

Comment:
This will break the link in the navbar. This can't be a static href, it's going to have to be something like
```pug
a(href="/profile/" + user.id)
```
in the view

I guess I was trying to start a pattern for routes and nav items, but it probably makes sense to break it here